core/src/test/java/org/apache/calcite/test/RelMetadataTest.java

```diff
@@ -788,6 +788,20 @@ final RelMetadataFixture sql(String sql) {
     sql(sql).assertThatRowCount(is(1D), is(0D), is(1D));
   }
 
+  @Test void testRowCountAggregateConstantKeysOnBigInput() {
```

Comment:
It's not obvious why this is an aggregate, I assume some rewrite step converts it into one.
That could use a comment.
Do you want to mention the JIRA case?

Reply:
+1

Reply:
I think it is arguably clear that the aggregate comes from the count (I'm just replicating the same patter of test method name and test code from the existing ones).
Added javadoc with Jira case.